### DISCUSSION TITLE
PLZ how to solve this situation

### DISCUSSION THREAD

**Discussion by fanhongyi-77** [PlayCover/PlayCover#189] [Q&A] [answered]
PLZ how to solve this situation:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

**Comment by ZhichGaming** [comment, marked as answer]
Run this in your terminal: 
`xcode-select --install`

**Reply by fanhongyi-77** [reply]
thx!
